docs/Github-workflow.md

```diff
@@ -0,0 +1,61 @@
+[Back to overview](https://github.com/WordPress/performance/blob/trunk/docs/README.md)
+
+# GitHub workflow for the Performance Lab plugin
+
+The Performance team uses GitHub to manage all code and related discussions for the Performance Lab plugin. Please follow the workflow below to ensure that all issues are properly tracked.
```

Comment:
The "Performance team" could have an external link to https://make.wordpress.org/core/2021/10/12/proposal-for-a-performance-team/